tests/components/refoss/test_config_flow.py

```diff
@@ -0,0 +1,50 @@
+"""Tests for the Refoss config flow."""
+from __future__ import annotations
+
+from refoss_ha.const import DOMAIN
+from refoss_ha.util import get_mac_address
+
+from homeassistant import config_entries, data_entry_flow
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+
+async def test_configured(hass: HomeAssistant):
+    """Test  configuration if refoss already configured."""
+
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == "form"
```

Comment:
Use the Enums here

Reply:
hello, May I ask how to test  'test_config_flow.py'  locally ?